recipes/superfile/recipe.yaml

```
@@ -0,0 +1,56 @@
+context:
+    version: "1.2.0.0"
+
+package:
+  name: superfile
+  version: ${{ version }}
+
+source:
+  url: https://github.com/yorukot/superfile/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 4f60bd87659708c8b572140487ee4767ea74da502f5546c521793a8c3b30480a
+
+build:
+  number: 0
+  script:
+    - if: unix
+      then:
+        - cd $SRC_DIR
+        - CGO_ENABLED=0 go build -o $PREFIX/bin/spf
```

The flag should already be set by the go package.
```suggestion
        - go build -o $PREFIX/bin/spf
```